### PR TITLE
Fixes for issue #1038

### PR DESCRIPTION
**Pull request by spficklin**
# Bug Fix

Issue #1038

## Description

There are very minor fixes prior to the Tripal v3.3 fix.

### CODE REVIEW

File: tripal/includes/TripalEntityCollection.inc [hunk 1]
```diff
@@ -528,6 +528,7 @@ class TripalEntityCollection {
     if ($job) {
       $job->setTotalItems($total_entities);
     }
+    $job->setInterval(1);
```
Comment (bradfordcondon) [inline tripal/includes/TripalEntityCollection.inc]: shouldnt this also be in the `if($job)` check?

Comment (spficklin) [reply tripal/includes/TripalEntityCollection.inc]: Oh good catch!  Thanks @bradfordcondon I fixed it.

**Comment by spficklin** [comment]
Approved by @laceysanderson via PMC slack message.  Merging.

**Review by bradfordcondon — COMMENTED** [review]
thanks stephen!

**Pull request merged**